# Android-specific fixes for inox-patchset/0001-fix-building-without-safebrowsing.patch.

--- a/chrome/browser/loader/chrome_resource_dispatcher_host_delegate.cc
+++ b/chrome/browser/loader/chrome_resource_dispatcher_host_delegate.cc
@@ -495,8 +495,6 @@ void ChromeResourceDispatcherHostDelegat
   // of the list, so one of them gets to decide if the resource is safe.
   content::ResourceThrottle* first_throttle = NULL;
 #if defined(OS_ANDROID)
-  first_throttle = DataReductionProxyResourceThrottle::MaybeCreate(
-      request, resource_context, resource_type, nullptr);
 #endif  // defined(OS_ANDROID)
 
 #if defined(SAFE_BROWSING_DB_LOCAL) || defined(SAFE_BROWSING_DB_REMOTE)
--- a/chrome/browser/safe_browsing/BUILD.gn
+++ b/chrome/browser/safe_browsing/BUILD.gn
@@ -12,6 +12,10 @@ proto_library("chunk_proto") {
 }
 
 static_library("safe_browsing") {
+  deps = [
+    "//chrome/common/safe_browsing:proto",
+  ]
+
   if (false) {
   sources = [
     "chrome_cleaner/chrome_cleaner_controller_impl_win.cc",
